src/rpc/mempool.cpp

```diff
@@ -854,15 +855,15 @@ static RPCHelpMan submitpackage()
             },
         },
         RPCExamples{
-            HelpExampleCli("testmempoolaccept", "[rawtx1, rawtx2]") +
-            HelpExampleCli("submitpackage", "[rawtx1, rawtx2]")
+            HelpExampleRpc("submitpackage", R"(["rawtx1", "rawtx2"])") +
+            HelpExampleCli("submitpackage", R"(["rawtx1", "rawtx2"])")
         },
         [&](const RPCHelpMan& self, const JSONRPCRequest& request) -> UniValue
         {
             const UniValue raw_transactions = request.params[0].get_array();
-            if (raw_transactions.size() < 1 || raw_transactions.size() > MAX_PACKAGE_COUNT) {
+            if (raw_transactions.size() > MAX_PACKAGE_COUNT) {
```

Comment:
Seems we are missing test coverage for this. Maybe add one to rpc_packages.py?

Reply:
Also, why delete the empty vector case? That should throw a JSONRPCError too, no?

Reply:
> Seems we are missing test coverage for this. Maybe add one to rpc_packages.py?

I'll have a look!

> Also, why delete the empty vector case? That should throw a JSONRPCError too, no?

We [throw](https://github.com/bitcoin/bitcoin/pull/29292/files#diff-9c5b83de6dc84af277e352c88b9291aa44340a3c75f572a0b51661eb0a838de9R879-R880) a `JSONRPCError` here if package size < 2, so all this should do is remove the confusing error message that the array needs to be at least size 1 (when actually it's size 2)

Reply:
Added test coverage: the [test diff](https://github.com/bitcoin/bitcoin/compare/6476ab562327d4690fddefc8336c3ddce211411a..021ecfea062151bce0d67bf1575fe7e5e1ee22db#diff-2757e99ad43064441a6401f5002232a6a311d92d16b49df66f51d427ce448a68) after changing the bounds checking shows how the error messages change

Reply:
thanks for adding a test :+1: